third_party/abseil-cpp/absl/synchronization/BUILD.gn

```diff
@@ -99,6 +102,7 @@ absl_test("waiter_test") {
     "//third_party/abseil-cpp/absl/random",
     "//third_party/abseil-cpp/absl/time",
   ]
+  } else { deps = [] }
```

Comment:
Does the absl_test template require `deps` to be set? If not, let's delete this, but if so, let's just format this block with newlines

Reply:
Yes, there is a `deps +=` somewhere that trips without this.

Reply:
To be specific, here:  //third_party/abseil-cpp/absl.gni:106:5: Undefined identifier.

Reply:
let's just add newlines here to format it like other blocks

```
} else {
  deps = []
}
```

Reply:
Done